server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -11,17 +11,17 @@
             </aside>
             <div class="app-card__content">
                 <header class="app-card__header govuk-!-margin-bottom-4">
-                    <h2 class="app-card__title" data-qa="{{moveType}}-title-{{loopIndex}}">           
-                        {% if user.isReceptionUser and 
-                            (
-                                ['COURT', 'CUSTODY_SUITE'].includes(arrival.fromLocationType) 
-                                and not arrival.isCurrentPrisoner
-                            ) 
-                        %}
-                            <a href="/prisoners/{{ arrival.id }}/confirm-arrival" class="govuk-link govuk-link--no-visited-state" data-qa="confirm-arrival"> {{ arrival.lastName }}, {{ arrival.firstName }}</a>
-                        {% else %}
-                            {{ arrival.lastName }}, {{ arrival.firstName }}
+                    <h2 class="app-card__title" data-qa="{{moveType}}-title-{{loopIndex}}">     
+                        {% if user.isReceptionUser %}
+                            {% if not arrival.isCurrentPrisoner %}
+                                    <a href="/prisoners/{{ arrival.id }}/confirm-arrival" class="govuk-link govuk-link--no-visited-state" data-qa="confirm-arrival"> {{ arrival.lastName }}, {{ arrival.firstName }}</a>
+                            {% elif arrival.fromLocationType === 'PRISON' %}
```

Comment:
If the from location is prison then isCurrentPrisoner is true by default - we won't have this field on transfers going forward so would be best  to do this check first